python/tests/sql/test_predicate.py

```diff
@@ -206,3 +206,13 @@ def test_st_overlaps(self):
         not_overlaps = self.spark.sql("select ST_Overlaps(c,d) from testtable")
         assert overlaps.take(1)[0][0]
         assert not not_overlaps.take(1)[0][0]
+
+    def test_st_ordering_equals_ok(self):
```

Comment:
Can you add a similar test for Spark SQL in Scala?

Reply:
Yes :)